### DISCUSSION TITLE
Bulk Audit

### DISCUSSION THREAD

**Discussion by danjneff** [snipe/snipe-it#13543] [Q&A]
I'm using version 6.1.0 of Snipe-IT as found on TurnkeyLinux. In an older version of Snipe-IT and a laptop not connected to a network I was able to use a barcode scanner to scan a bunch of assets into a .txt file. I was then able to copy the .txt to a PC connected to the network with Snipe-IT. I could then copy and paste all of the assets in the .txt into the "Asset Tag" field of the "Bulk Audit" page pick a location and it would perform the audit on all the items. With the new version I can only enter one item into the "Asset Tag" field at a time. Screenshots of old one attached. The new version submits every time I hit the enter key. What blade do I need to change and how do I change the "Asset Tag" field so it can accept a number of assets and audit them all?

![BulkAuditEntry](https://github.com/snipe/snipe-it/assets/33243477/7802e355-64cf-4502-b81a-9db06f5f54db)
![BulkAuditResult](https://github.com/snipe/snipe-it/assets/33243477/301724e5-7b25-47bf-88c4-ffe7381582b0)

**Comment by amrit647** [comment]
Same issue here.
